_docs/usecases/kubernetes/backups.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: guide
+title: StorageOS Docs - Backups
+anchor: usecases
+module: usecases/kubernetes/backups
+---
+
+# File exfiltration on Kubernetes from StorageOS persistent volumes
+
+This example is provided to give ideas of how to exfiltrate files that have
+been written to a StorageOS persistent volume. This example uses nginx, rsync
+and sftp sidecars in an application pod. This approach may be useful to create
```

Comment:
I think this is confusing. I don't like the fact that we are shipping 3 side cars in one Pod. Why not 3 different yamls, one for each example with the same 'main' container and one of the sidecars. 
We can create a list of 3 examples. aka
- Web server
example description, etc
- Rsync
example description, etc
- SFTP
example description, etc

I'd say: 
In this example, we provide three strategies to access the files that a application wrote to a volume. First you can export the files by using a Nginx as a web file server, it is also possible to transfer files using rsync and finally you can access them via SFTP.

Reply:
I think this 2 paragraphs should be different. Add a section about "Context" and explain what do you have in the example. The explanation is too dense and I believe is confusing unless I go to the yamls and checkout what is in there.

Reply:
I would change the following section:

This example is provided to give ideas of how to exfiltrate files that have been written to a StorageOS persistent volume. This example uses nginx, rsync and sftp sidecars in an application pod. This approach may be useful to create out-of-cluster backups of data.

In this example the “application” container is the container “main”, which has rsync, nginx and sftp sidecar containers. The StorageOS volume that the application is writing to will be mounted into the sidecar containers as well so backup files etc. can be exfiltrated.

The files create a stateful set that can be used AFTER a StorageOS cluster has been created. For more information on how to install a StorageOS cluster please see [the StorageOS documentation] (https://docs.storageos.com/docs/introduction/quickstart).

for: 

This example shows different options to access files that have been written to a StorageOS persistent volume previously. 
The access of files is achieved by adding side car containers to the `main` container that shared the same filesystem space. The `main` container is considered as the generator of data, for instance a database application. 

In this example, we provide three strategies to access the files that a application wrote to a volume. First you can export the files by using a Nginx as a web file server, it is also possible to transfer files using rsync and finally you can access them via SFTP.
This approach might be of interest to visualise files or export backups generated in the filesystem of the Pod.

1. Clone repo

2. Deploy example
* Web server
  1 Description
  2 Yamls
  3 Outputs of examples

* Rsync
  1 Description
  2 Yamls
  3 Outputs of examples

* SFTP
  1 Description
  2 Yamls
  3 Outputs of examples